### PR TITLE
Encrypted transmission of login password

### LINKED ISSUE #1300: Optimize the transmission of console login password plaintext field
## Demand advice

Please answer these questions before submitting them. Thank you.

### Is your function request related to the problem?

Yes.

### Describe the function you want

Optimize the transmission of console login password plaintext field.

### PR DESCRIPTION
**Pull request by HKMV**
Fixes #1300

Changes proposed in this pull request:
-  Modify console web-ui user module, add crypto and buffer lib
-  Modify console server static page
-  Modify server auth module write AESUtil
-  Modify server auth module jwt auth filter

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: threadpool/server/auth/src/main/java/cn/hippo4j/auth/toolkit/AESUtil.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import javax.crypto.*;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+/**
+ * Aes加解密算法工具类
```
Comment (magestacks) [inline threadpool/server/auth/src/main/java/cn/hippo4j/auth/toolkit/AESUtil.java]: Please use English for comments in the code

File: threadpool/server/console/src/main/resources/static/index.html [hunk 2]
```diff
@@ -1 +1 @@
...
...+")");r.type=t,r.request=a,n[1](r)}u[e]=void 0}};var k=setTimeout((function(){d({type:"timeout",target:o})}),12e4);o.onerror=o.onload=d,document.head.appendChild(o)}return Promise.all(c)},h.m=e,h.c=t,h.d=function(e,c,n){h.o(e,c)||Object.defineProperty(e,c,{enumerable:!0,get:n})},h.r=function(e){"undefined"!==typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},h.t=function(e,c){if(1&c&&(e=h(e)),8&c)return e;if(4&c&&"object"===typeof e&&e&&e.__esModule)return e;var n=Object.create(null);if(h.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:e}),2&c&&"string"!=typeof e)for(var t in e)h.d(n,t,function(c){return e[c]}.bind(null,t));return n},h.n=function(e){var c=e&&e.__esModule?function(){return e["default"]}:function(){return e};return h.d(c,"a",c),c},h.o=function(e,c){return Object.prototype.hasOwnProperty.call(e,c)},h.p="/",h.oe=function(e){throw console.error(e),e};var d=window["webpackJsonp"]=window["webpackJsonp"]||[],o=d.push.bind(d);d.push=c,d=d.slice();for(var k=0;k<d.length;k++)c(d[k]);var i=o;n()})([]);</script><script src=/static/js/app.e0306130.js></script></body></html>
```
Comment (magestacks) [inline threadpool/server/console/src/main/resources/static/index.html]: static I will unify the changes later, and this PR can delete the changes

File: threadpool/server/auth/src/main/java/cn/hippo4j/auth/toolkit/AESUtil.java (hunk 1 above)
Comment (HKMV) [reply threadpool/server/auth/src/main/java/cn/hippo4j/auth/toolkit/AESUtil.java]: ok

File: threadpool/server/console/src/main/resources/static/index.html (hunk 2 above)
Comment (HKMV) [reply threadpool/server/console/src/main/resources/static/index.html]: restored

**Review by magestacks — APPROVED** [review]

**Pull request merged**